src/datasets-context-json.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright (C) 2024 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+/**
+ * \file
+ *
+ * \author Eric Leblond <el@stamus-networks.com>
+ */
+
+#ifndef SURICATA_DATASETS_CONTEXT_JSON_H
+#define SURICATA_DATASETS_CONTEXT_JSON_H
+
+#include <suricata-common.h>
```

Comment:
Should be `"suricata-common.h"`.  If a user has the headers from another version installed, they'll be picked up instead of the headers in the source directory.

Reply:
wonder if we can catch this using a tool